src/convertToObject.js

```diff
@@ -1,18 +1,30 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  let arrayFromString = sourceString.trim().split(';');
+
+  arrayFromString = arrayFromString.map(line => line.replace(/ /g, ''));
+  arrayFromString = arrayFromString.map(line => line.replace('\n', ''));
+  arrayFromString = arrayFromString.map(line => line.replace('\n\n', ''));
+
+  arrayFromString = arrayFromString.filter(
+    line => line !== '' && line !== '\n');
+
+  arrayFromString = arrayFromString.map(
+    line => line.replace('.2s', ' .2s '));
+
+  arrayFromString = arrayFromString.map(
+    line => line.replace('solid', ' solid '));
+  arrayFromString = arrayFromString.map(line => line.replace('!', ' !'));
```

Comment:
What if the styles have different values than `.2s` or `solid`? Will your code still work? It is not necessary to do it this way.

Btw. you are iterating through the array way more times than it's necessary. Try to find a simpler solution. I'm sure you are gonna solve with way less steps ;)